### PR TITLE
[-] Models Introspection - Support nested custom schemas for models

### PR DESCRIPTION
**Pull request by lucasscariot**
## Pull Request checklist:

- [x] Write an explicit title for the Pull Request
- [x] Write changes made in the CHANGELOG.md
- [x] Test manually the implemented changes
- [x] Review my own code (indentation, syntax, style, simplicity, readability)
- [x] Wonder if you can improve the existing code

### CODE REVIEW

**Comment by arnaudbesnier** [comment]
Fix: https://github.com/ForestAdmin/forest-express-mongoose/issues/76

**Comment by lucasscariot** [comment]
[(?) Mongoose users want nested custom schema support](https://trello.com/c/TwuSIowQ/1730-mongoose-users-want-nested-custom-schema-support)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Change Log
 
 ## [Unreleased]
+### Updated
+- Apimap - Send embedded schemas.
```
Comment (arnaudbesnier) [inline CHANGELOG.md]: Models Introspection - Support nested custom schemas for models.

File: adapters/mongoose.js [hunk 2]
```diff
@@ -131,6 +131,10 @@ module.exports = function (model, opts) {
     } else if (opts.instance === 'ObjectID') {
       // Deal with ObjectID
       return 'String';
+    } else if (opts.instance === 'Embedded') {
+      return objectType(opts.schema.obj, function (fieldName) {
+        return getTypeFromNative(opts.schema.obj[fieldName].type);
+      });
```
Comment (arnaudbesnier) [inline adapters/mongoose.js]: Does not work properly with this code:

```
var bluSchema = new mongoose.Schema({
    a: String,
    b: Boolean,
    c: Date,
    d: Number,
    e: { type: String, enum: ['abababa', 'b', 'c'] }
});

blu1: bluSchema,
blu2: {
    a: String,
    b: Boolean,
    c: Date,
    d: Number,
    e: { type: String, enum: ['abababa', 'b', 'c'] }
  },
```
`blu1` and `blu2` should behave the same. It does not seem to be the same.

**Pull request merged**